### DISCUSSION TITLE
Not getting xiphos repository

### DISCUSSION THREAD

**Discussion by nDR01d** [crosswire/xiphos#1112] [Q&A]
I have tried with Xiphos 4.2.1 (download from PPA), Bibletime (flatpack and synaptic) to get access to the xiphos repo with no luck at all. It just times out and shows literally nothing on the modules page. I'm using ftp.xiphos.org /pub/xiphos 20090514005900. I have also tried using ftp via FileZilla. Have asked on chat but no replies. Any suggestions please?
Edit: Should have mentioned I'm using Kernel: 5.4.0-109-generic x86_64 bits: 64 compiler: gcc v: 9.4.0 Desktop: Cinnamon 5.2.7 wm: muffin dm: LightDM Distro: Linux Mint 20.3 Una. Hope this helps.

**Comment by LAfricain** [comment]
Just try later. It happens sometimes, nothing to see with your version of xiphos.
Which module do yo like to download?

**Reply by nDR01d** [reply]
I have been trying it every few days. I also just tried Bishop on my android phone (Samsung Galaxy S20 FE) and all of the other repos (crosswire, Lockman, etc) worked except for xiphos, which timed out. I'm interested in downloading (according to the info from https://xiphos.org/links/) Strongs (real) Hebrew and Greek dictionaries, Inverted Strongs Greek and Hebrew dictionaries and the one-year Bible reading plan.

**Comment by LAfricain** [comment]
> Strongs (real) Hebrew and Greek dictionaries

You can dowload this modules in the Crosswire beta repo.
Your problem is really strange. I have no problem with the Xiphos repository. Did you test in an other place (with an other internet connection)?

**Reply by nDR01d** [reply]
Yes, I did try it on my mobile phone using the Bishop app. Same thing happened ie: time out. Maybe Australia has a domain block?! I've got no idea whats going on.

**Comment by karlkleinpaste** [comment]
the server has an extensive iptables firewall, made necessary since years ago, due to rampant abuse.
what is your externally visible IP address? the fw may need adjustment.

**Reply by nDR01d** [reply]
According to whatismyipadress.com IPv6: [2001:8004:5100:28e5:6dda:21c9:d134:6e47] IPv4: [1.145.50.103]
and via bash command "ip address" wlo1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    link/ether 5c:3a:45:53:fb:3d brd ff:ff:ff:ff:ff:ff
    inet 192.168.1.34/24 brd 192.168.1.255 scope global dynamic noprefixroute wlo1
       valid_lft 42974sec preferred_lft 42974sec
    inet6 2001:8004:5100:28e5:6dda:21c9:d134:6e47/64 scope global temporary dynamic 
       valid_lft 604577sec preferred_lft 85815sec
    inet6 2001:8004:5100:28e5:7471:c6a6:e0df:3ba9/64 scope global mngtmpaddr noprefixroute 
       valid_lft forever preferred_lft forever
    inet6 fe80::bae7:b694:e045:cf9e/64 scope link noprefixroute

I hope I've covered everything. I live rurally in NSW, so the internet is via mobile phone towers. If you need further clarification, let me know.

**Comment by karlkleinpaste** [comment]
try again.

**Reply by nDR01d** [reply]
It works! Thank you.
